test/conftest.py

```diff
@@ -4,6 +4,7 @@
 Copyright (c) 2016 Ohmu Ltd
 See LICENSE for details
 """
+from distutils.version import LooseVersion
```

Comment:
Cf. https://github.com/aiven/pglookout/pull/82 It's probably better to not reintroduce deprecated API usage.